independent-publisher-connectors/PaySpace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4476 @@
+                      "TradeUnion": {
+                        "type": "string",
+                        "description": "TradeUnion"
+                      },
+                      "IsPromotion": {
+                        "type": "boolean",
+                        "description": "IsPromotion"
+                      },
+                      "Roster": {
+                        "type": "string",
+                        "description": "Roster"
+                      },
+                      "Job": {
+                        "type": "string",
+                        "description": "Job"
+                      },
+                      "Comments": {
+                        "type": "string",
+                        "description": "Comments"
+                      },
+                      "AltPositionName": {
+                        "type": "string",
+                        "description": "AltPositionName"
+                      },
+                      "DateAdded": {
+                        "type": "string",
+                        "description": "DateAdded"
+                      },
+                      "PositionEffectiveDate": {
+                        "type": "string",
+                        "description": "PositionEffectiveDate"
+                      },
+                      "CustomTradeUnion": {
+                        "type": "string",
+                        "description": "CustomTradeUnion"
+                      },
+                      "CustomFields": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "Code": {
+                              "type": "string",
+                              "description": "Code"
+                            },
+                            "Label": {
+                              "type": "string",
+                              "description": "Label"
+                            },
+                            "Value": {
+                              "type": "string",
+                              "description": "Value"
+                            }
+                          }
+                        },
+                        "description": "CustomFields"
+                      }
+                    }
+                  },
+                  "description": "value"
+                }
+              }
+            }
+          }
+        }
+      },
+      "patch": {
+        "summary": "Update a single position record",
+        "description": "Update a single employee position record based on the specified EmployeePositionId.",
+        "operationId": "UpdateASinglePositionRecord",
+        "parameters": [
+          {
+            "name": "EmployeePositionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "{{EmployeePositionId}} The EmployeePositionId"
```

Comment:
@mint-virgil The 'x-ms-summary' property is required.

Reply:
updated in the latest commit